### DISCUSSION TITLE
Opacity for nodes and link with 2D graph

### DISCUSSION THREAD

**Discussion by JacobHamelMottiez** [vasturiano/react-force-graph#533] [Q&A] [answered]
Hi, 

I have the following graph rendered with react-force-graph : 
![image](https://github.com/user-attachments/assets/e897eb87-32a5-48a1-854f-d9dfcc8d7a22)

I hope you are all doing well! I am currently trying to render a graph in 2D and I would like to adjust the opacity of the link and the nodes : 
![image](https://github.com/user-attachments/assets/2fa133db-1d1a-4a8e-8c82-2a31bf67881c)

Could somebody help me a bit? 

Thanks,
Jacob

**Comment by vasturiano** [comment, marked as answer]
@JacobHamelMottiez you can use the `nodeColor`/`linkColor` props to set the opacities via the alpha channel.

For example:
```js
linkColor={() => "rgba(255, 0, 0, 0.35)"}
```

**Reply by JacobHamelMottiez** [reply]
Hi @vasturiano! Thanks for your answer, it helps very much :)
